runtime/pack/dist/opt/netrw/autoload/netrw.vim

```diff
@@ -19,23 +19,7 @@ if &cp || exists("g:loaded_netrw")
   finish
 endif
 
-" Check that vim has patches that netrw requires.
-" Patches needed for v7.4: 1557, and 213.
-" (netrw will benefit from vim's having patch#656, too)
-let s:needspatches=[1557,213]
-if exists("s:needspatches")
-  for ptch in s:needspatches
-    if v:version < 704 || (v:version == 704 && !has("patch".ptch))
-      if !exists("s:needpatch{ptch}")
-        unsilent echomsg "***sorry*** this version of netrw requires vim v7.4 with patch#".ptch
-      endif
-      let s:needpatch{ptch}= 1
-      finish
-    endif
-  endfor
-endif
-
-let g:loaded_netrw = "v175"
+let g:loaded_netrw = "v176"
```

Comment:
What version are you aiming to support here please? I would say to keep a version check for your target version, just in case someone is trying to download runtime files from Vim and copies to an older version.

Reply:
> What version are you aiming to support here please?

I currently targeting the latest vim tag and nightly neovim. This is done so I can decrostify a lot of the old cruft in netrw.

> I would say to keep a version check for your target version, just in case someone is trying to download runtime files from Vim and copies to an older version.

I don't know how I would go about this... Do you think is a common problem? Should I perhaps pin the tag most recent to the last snapshot that has been merged?

Reply:
Okay, for now, I'd add a version check for e.g. Vim 9.1. And If you are a later relying on specific features that is only available with a later Vim version, I'd got with specific feature checks, like `has("feature")`.

Reply:
> I'd add a version check for e.g. Vim 9.1

Ok, is the latest runtime available in vim 9.1? I currently using `dist#vim9#{Open, Launch}` that have been merged with #16494

> And If you are a later relying on specific features that is only available with a later Vim version, I'd got with specific feature checks, like has("feature").

Yes, specific features that are optional in building vim are behind a check

Reply:
ah okay, then we should probably make it depend on v9.1.1054 (which has `dist#vim9#Open()`)